src/index.html

```diff
@@ -5,8 +5,368 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+
+  <body class="page">
+    <header class="header">
+      <a href="#home" class="header_logo">
+        <img src="images/logo.png" alt="Moyo logo">
+      </a>
+
+      <nav class="header__nav nav">
+        <ul class="nav_list">
+          <li class="nav_item nav_item--linked">
+            <a href="#apple" class="nav_link nav_link--is-active">Apple</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#samsung" class="nav_link">Samsung</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#smartphones" class="nav_link">Smartphones</a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#laptops"
+              class="nav_link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#gadgets" class="nav_link">Gadgets</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#tablets" class="nav_link">Tablets</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#photo" class="nav_link">Photo</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#video" class="nav_link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__container">
+        <div class="card card--content" data-qa="card">
+          <img
+            class="card__photo"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+          >
+
+          <h1 class="card__tittle card__tittle--margins">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
```

Comment:
h1 can be only one at page

Reply:
why do you need classes with ...--margins?
add margin to class card__tittle and other